app/src/main/res/layout/fragment_details.xml

```diff
@@ -36,8 +36,7 @@
             android:layout_marginTop="@dimen/margin_top_fragment_image_button"
             android:layout_marginEnd="@dimen/margin_end_fragment_details_icon_favorites"
             android:background="@color/white_gray"
-            android:onClick="@{(view) -> viewModel.addToMealFavorites(view)}"
```

Comment:
Any reason for removing this and setting the clickListener inside the fragment?

Reply:
I changed back to the previous approach.